data/campaigns/Dead_Water/scenarios/10_The_Flaming_Sword.cfg

```diff
@@ -288,7 +288,7 @@
         [/message]
         [message]
             speaker=Caladon
-            message= _ "Well, he calls himself Mal-Govon now, but yes. He called this land the KINGDOM of Agnovon. He was its first king. And its last, and every one in between, too! He kept getting older and older, but he never died. Well, NOW he’s dead of course. He still putters around in that castle over there, but he hasn’t really cared about keeping his kingdom under control for a good long while. I bet I can wake him up, though. Watch THIS!"
+            message= _ "Well, he calls himself Mal-Govon now, but yes. He called this land the KINGDOM of Agnovon. He was its first king. And its last. And each one in between too! He kept getting older and older, but he never died. Well, NOW he’s dead of course. He still putters around in that castle over there, but he hasn’t really cared about keeping his kingdom under control for a good long while. I bet I can wake him up, though. Watch THIS!"
```

Comment:
I don't think sentences should begin with "And", as I have been lectured countless times in IELTS class.

Reply:
This is flavor. It's like when you see sentences such as:
Jordan. Is. The. GOAT.
They are pauses with extra emphasis and affects the way it's read. Note, that the original has a sentence beginning with 'And', and I repeated the pattern for the next clause.